src/memary/agent/base_agent.py

```diff
@@ -145,10 +145,11 @@ def load_llm_model(self, llm_model_name):
             self.openai_api_key = os.environ["OPENAI_API_KEY"]
             self.model_endpoint = "https://api.openai.com/v1"
             self.llm = OpenAI(model="gpt-3.5-turbo-instruct")
-        elif llm_model_name == "llama3":
-            self.llm = Ollama(model="llama3", request_timeout=60.0)
         else:
-            raise ("Please provide a proper llm_model_name.")
+            try:
+                self.llm = Ollama(model=llm_model_name, request_timeout=60.0)
```

Comment:
Can't go directly down this path since the `llm_model_name` won't be "ollama3" but can be any of the llm models that can run on ollama like "llama3" or "mistral". This just ensures that as long as the model is valid and pulled into the user's local ollama, no exception is thrown.